src/main/webapp/script.js

```diff
@@ -42,7 +42,9 @@ async function getYouTubeComments() {
 
 /**
  * Fetches data and adds to html
- */
+  * Credit for RegEx method derived from top answer on: 
+  * https://stackoverflow.com/questions/28735459/how-to-validate-youtube-url-in-client-side-in-text-box
```

Comment:
Can we remove this regex comment here? It seems out of place since it's also found w/ the `extraYouTubeUrl` function.